dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -73,30 +74,37 @@ function Catalog(props: ICatalogProps) {
       isFetching,
       selected: { error },
       items: charts,
+      categories,
     },
     fetchCharts,
     cluster,
     namespace,
+    fetchChartCategories,
     getCSVs,
     csvs,
     repo,
     filter: propsFilter,
   } = props;
   const dispatch = useDispatch();
   const [filters, setFilters] = useState(initialFilterState());
+  const initialRepoValue = propsFilter[filterNames.REPO]
+    ? String(propsFilter[filterNames.REPO])
+    : "";
+  const [currentRepo, setCurrentRepo] = useState(initialRepoValue);
```

Comment:
same, you don't need this code, no?

Reply:
Not in this PR, but the point is that `repo` property is undefined, so just created this prop in the component's state. It will be useful for filtering charts from a specific repo, but to keep this PR as simple as possible, I'm temporarily removing this code.